Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-stretch
+FROM golang:1.14-stretch
```

Comment:
Why 1.14?

Reply:
Either 1.13 or 1.14 should work just fine. As 1.13 will eventually be unsupported, I'm jumping straight to 1.14. Can use 1.13 as well here, let me know if you want it that way.

Ideally we should merge https://github.com/opencontainers/runc/pull/2239 first and just rebase this one on top of it, but it's still marked as draft by @thaJeztah.

Reply:
1.14 has breaking changes on `SIGURG`&`EINTR` stuff  https://golang.org/doc/go1.14#runtime

Reply:
It was marked "draft", because I separated it from https://github.com/opencontainers/runc/pull/2234. Hopefully that one could be merged soon, and I'll rebase https://github.com/opencontainers/runc/pull/2239